src/views/List.jsx

```diff
@@ -10,10 +30,31 @@ export function List({ data, listToken }) {
 			{data.length === 0 && (
 				<p>Your shopping list is empty. Click on "Add item" to begin!</p>
 			)}
-			<ul>
-				{data.map((item) => (
-					<ListItem key={item.id} name={item.name} />
-				))}
+			<form onSubmit={(e) => e.preventDefault()}>
+				<label htmlFor="filter">Filter items</label>
+				<br />
+				<input
+					type="text"
+					name="filter"
+					id="filter"
+					value={inputItem}
+					onChange={handleInput}
+					placeholder="Start typing here..."
+				/>
+				<Button
+					ariaLabel={'clear input field'}
+					label={'x'}
+					type={'reset'}
+					onClick={handleClear}
+				/>
+			</form>
```

Comment:
Great use of semantic html for the form elements! So much easy to implement a11y features this way!